elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -178,6 +178,13 @@
             <version>2.5.0</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.ant</groupId>
```

Comment:
Why do we need ant?

Reply:
Used for read in graphql expected response from json file.

Reply:
I'd rather not add a dependency on ant in Elide.   There are other places in Elide where we slurp in files.  Let's just do the same approach.